src/main/java/io/github/programminglife2016/pl1_2016/Launcher.java

```diff
@@ -26,13 +26,15 @@ private Launcher() {
     public static void main(String[] args) throws IOException {
         System.out.println("Started loading.");
         long startTime = System.nanoTime();
-        InputStream is = Launcher.class.getResourceAsStream("/genomes/testGraph.gfa");
-        NodeCollection nodeCollection = new SegmentParser().parse(is);
+        InputStream is = Launcher.class.getResourceAsStream("/genomes/TB328.gfa");
+        InputStream positions = Launcher.class.getResourceAsStream("/genomes/TB328.dot");
+        NodeCollection nodeCollection = new SegmentParser(positions).parse(is);
         long endTime = System.nanoTime();
         System.out.println(String.format("Loading time: %f s.", (endTime - startTime)
                 / NANOSECONDS_PER_SECOND));
-        BubbleDispatcher dispatcher = new BubbleDispatcher(nodeCollection);
-        Server server = new RestServer(dispatcher.getLevelBubbles(0, BUBBLE_THRESHOLD));
```

Comment:
You can remove the commented code. This can later be retrieved from the bubble collapser